framework/web/Request.php

```diff
@@ -89,10 +89,6 @@ class Request extends \yii\base\Request
      * The name of the HTTP header for sending CSRF token.
      */
     const CSRF_HEADER = 'X-CSRF-Token';
-    /**
-     * The length of the CSRF token mask.
-     */
-    const CSRF_MASK_LENGTH = 8;
```

Comment:
You should deprecate this instead of removing - this is public constant, people may use it.